Doc/library/stdtypes.rst

```diff
@@ -3996,10 +3996,7 @@ The constructors for both classes work the same:
    argument.
 
    Note, the *elem* argument to the :meth:`__contains__`, :meth:`remove`, and
-   :meth:`discard` methods may be a set.  To support searching for an equivalent
-   frozenset, the *elem* set is temporarily mutated during the search and then
-   restored.  During the search, the *elem* set should not be read or mutated
-   since it does not have a meaningful value.
+   :meth:`discard` methods may be a set.
```

Comment:
While correct, this leaves the mechanism as a mystery.   There should probably be a sentence saying that a new temporary frozenset is built from *elem*.  That would be a minor implementation detail except for the O(n) conversion cost.